endorsement/storeapi.proto

```diff
@@ -42,7 +42,7 @@ message GetEndorsementsArgs{
 }
 
 message Response {
-    int32 error_value = 1;
+    bool result = 1;
     string error_detail = 2;
```

Comment:
I wonder if we should define a top-level Status message that we can extend independently